events/wmt.md

```diff
@@ -129,45 +129,52 @@ Some people have been organisers over many years:
 
 ### Average score and average z-score
 
-For the **average score**, human assessment scores for translations are standardised according to each human assessor's overall mean and standard deviation.  Then a system-level score is computed.
+For the **average score**, human assessment scores for translations are standardised according to each human assessor's overall mean and standard deviation. Then a system-level score is computed.
```

Comment:
Hi @liashahnazaryan!
Please remember that we separate sentences with an `enter`, following our [style guide](https://machinetranslate.org/style#spacing).
Let me know if you have any question.

Reply:
Done. Don't really know how that happened this time :)